spec/models/contact_spec.rb

```diff
@@ -1,10 +1,146 @@
 require 'rails_helper'
 
 RSpec.describe Contact, type: :model do
-  describe "validations" do
-  it { should validate_presence_of(:first_name) }
-  it { should validate_presence_of(:last_name) }
+  describe "associations" do
+    it { should belong_to(:user) }
+    it { should belong_to(:company).optional }
```

Comment:
I like the use of .optional, did not know that piece of code. I learn something new every time I look over one of your PRs.

Reply:
Thanks!